pkgs/development/python-modules/albumentations/default.nix

```diff
@@ -29,6 +29,17 @@
   torchvision,
 }:
 
+let
+  albucore' = albucore.overridePythonAttrs rec {
```

Comment:
Overriding Python dependencies is not allowed within `python3Packages`.

Reply:
Alright, I'll swap this to a revert of the bump and hopefully it doesn't just get broken again in the next mass update